### PR TITLE
Fix panic when running command without resource

### PR DESCRIPTION
**Pull request by thijsdejong**
##### ISSUE TYPE

 - Bug fix Pull Request

##### SUMMARY

This fixes a panic when running a command with an allowed verb but without a resource. For example: `@Botkube get`.

### CODE REVIEW

**Comment by PrasadG193** [comment]
Hey @thijsdejong, thanks for fixing this. Do you mind adding test for this in our e2e test suite?  https://github.com/infracloudio/botkube/blob/develop/test/e2e/command/kubectl.go#L46

**Comment by thijsdejong** [comment]
> Hey @thijsdejong, thanks for fixing this. Do you mind adding test for this in our e2e test suite? https://github.com/infracloudio/botkube/blob/develop/test/e2e/command/kubectl.go#L46

I added the test :shipit:

**Review by PrasadG193 — APPROVED** [review]

**Comment by PrasadG193** [comment]
@thijsdejong CI failures are blocking merge. Could you please take a look?

File: test/e2e/command/kubectl.go [hunk 1]
```diff
@@ -64,6 +64,11 @@ func (c *context) testKubectlCommand(t *testing.T) {
 			expected: "```\nCommand not supported. Please run /botkubehelp to see supported commands.\n```",
 			channel:  c.Config.Communications.Slack.Channel,
 		},
+		"kubectl command without a resource": {
+			command:  "get",
+			expected: "```Command not supported. Please run /botkubehelp to see supported commands.```",
```
Comment (PrasadG193) [inline test/e2e/command/kubectl.go:69]:
```suggestion
			expected: "```\nCommand not supported. Please run /botkubehelp to see supported commands.\n```",
```

**Pull request merged**